src/webview/direct-connect-form.html

```diff
@@ -16,10 +16,11 @@ <h1>Set up a Direct Connection</h1>
       <template data-if="this.errorMessage()">
         <div class="info error" data-text="this.errorMessage()"></div>
       </template>
+
       <form data-on-submit="this.handleSubmit(event)">
         <div class="input-container">
           <div>
-            <label for="connection-label" class="label">Connection Label (optional)</label>
```

Comment:
should this be required?

Reply:
Nope! Fixed in https://github.com/confluentinc/vscode/pull/636/commits/b33b3d2c1a14523e9ea6ad19da5fb6926c0f4f3c after some merge conflict cleanup